### PR TITLE
Update AlertBox colors

### PR DESCRIPTION
**Pull request by lukaw3d**
https://www.figma.com/file/G6EVdJhfGbDBZRdpWnUyC7/Wallet-Design-System?type=design&node-id=22-1429&t=E44uYOZqmDSuC2xp-0

| Before | After |
| --- | --- |
| image  | image |
|  a1 | localhost_3000_account_oasis1qrdx0n90qz6d2me83gwdykgfu233x8f6psa4l3vd |
| a2 | localhost_3000_account_oasis1qrdx0n90qz6d2me83gwdykgfu233x8f6psa4l3vd (1) |

### CODE REVIEW

**Comment by lukaw3d** [comment]
https://www.figma.com/file/G6EVdJhfGbDBZRdpWnUyC7/Wallet-Design-System?type=design&node-id=22-1429&t=E44uYOZqmDSuC2xp-0

File: src/styles/theme/ThemeProvider.tsx [hunk 1]
```diff
@@ -53,17 +53,46 @@ const grommetCustomTheme: ThemeType = {
...
+        dark: '#2ad5ab',
+        light: '#c2f5ea',
+      },
+      'alert-box-ok-background': {
+        dark: '#c2f5ea',
+        light: '#2ad5ab',
+      },
+      'alert-box-warning': {
+        dark: '#f3d45e',
+        light: '#f0e8cb',
```
Comment (buberdds) [inline src/styles/theme/ThemeProvider.tsx:81]: `#F3D45E`

**Review by buberdds — COMMENTED** [review]
Is there an issue with bg colors or am I looking at wrong Firma?

File: src/styles/theme/ThemeProvider.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/styles/theme/ThemeProvider.tsx:81]: I think that's a bug on mocks - border `#F3D45E` should match the icon `#f0e8cb`

File: src/styles/theme/ThemeProvider.tsx [hunk 2]
```diff
@@ -53,17 +53,46 @@ const grommetCustomTheme: ThemeType = {
...
+      },
+      'alert-box-warning-background': {
+        dark: '#f0e8cb',
+        light: '#f3d45e',
+      },
+      'alert-box-error': {
+        dark: '#d24c00',
```
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/styles/theme/ThemeProvider.tsx:88]: oops, seems to have changed

File: src/styles/theme/ThemeProvider.tsx [hunk 3]
```diff
@@ -53,17 +53,46 @@ const grommetCustomTheme: ThemeType = {
...
+        light: '#ffe7d9',
+      },
+      'alert-box-error-background': {
+        dark: '#ffe7d9',
+        light: '#f26111',
```
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/styles/theme/ThemeProvider.tsx:93]: Let's do that in a separate PR. I think the font color change applies everywhere

**Review by buberdds — APPROVED** [review]

**Pull request merged**